index.html

```diff
@@ -21,22 +21,22 @@
         const sprintsInAnIteration = 6;
         
         function DetermineTriagerGivenDate(date){
-            const arbitraryMondayAtMidnight = new Date('July 3, 2023 00:00:00');
+            const arbitraryMondayAtMidnight = new Date('Novemember 20, 2023 00:00:00');
             const weeksSinceArbitraryMonday = Math.floor((date - arbitraryMondayAtMidnight) / millisecondsInAWeek);
             
-            const triagers = ["JaM", "KD", "MS", "SS", "EH", "JO"];
+            const triagers = ["JaM", "KD", "MS", "SS", "JM", "MC", "AE", "JO"];
             return triagers[weeksSinceArbitraryMonday % triagers.length];
         }
 
         function DetermineDemoerGivenDate(date){
-            const arbitraryFirstDemoOfIteration = new Date('June 12, 2023 18:30:00'); // O1 Demo
+            const arbitraryFirstDemoOfIteration = new Date('January 1, 2024 18:30:00'); // O1 Demo

```

Comment:
Nice to see someone else committing to this repo! 😀

Changes all look good, except a minor change is needed:

```suggestion
            const arbitraryFirstDemoOfIteration = new Date('December 18, 2023 18:30:00'); // Q1 Demo
```

The logic relies on `arbitraryFirstDemoOfIteration` being the first demo of an iteration. This lines things up to account for the 3 week sprint at the end of the iteration. Things would have been much simpler if all the sprints were two weeks 🙃

Reply:
Updated! Haha agreed 😂 I also changed the order of the array to keep Ken as the next person on demo duty!